doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -43,102 +46,121 @@ box.info.synchro
+
+        -   ``confirm_lag`` (since version :doc:`3.2.0 </release/3.2.0>`) -- the time in seconds that the latest successfully
+            confirmed entry waited for the quorum to collect.
+
     *   ``quorum`` -- the resulting value of the
         :ref:`replication_synchro_quorum <cfg_replication-replication_synchro_quorum>` configuration option.
         Since version :doc:`2.5.3 </release/2.5.3>`, the option can be set as a dynamic formula.
         In this case, the value of the ``quorum`` member depends on the current number of replicas.
 
     **Example 1:**
 
-    In this example, the ``quorum`` field is equal to ``1``.
+    In this example, the ``quorum`` field is equal to `1`.
     That is, synchronous transactions work like asynchronous ones.
     `1` means that a successful WAL writing to the master is enough to commit.
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.info.synchro
+        instance001> box.info.synchro
         ---
         - queue:
             owner: 1
+            confirm_lag: 0
             term: 2
+            age: 0
             len: 0
             busy: false
           quorum: 1
         ...
 
     **Example 2:**
 
-    First, set a quorum number and a timeout for synchronous replication using the following command:
+    First, set a quorum number and a timeout for synchronous replication in the configuration file (``config.yaml``):
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: yaml
 
-        tarantool> box.cfg{
-                 > replication_synchro_quorum=2,
-                 > replication_synchro_timeout=1000
-                 > }
+        replication:
+          synchro_quorum: 2
+          synchro_timeout: 1000
 
-    Next, check the current state of synchronous replication:
+    Check the current state of synchronous replication:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> box.info.synchro
+        app:instance001> box.info.synchro
         ---
         - queue:
             owner: 1
+            confirm_lag: 0
             term: 2
+            age: 0
             len: 0
             busy: false
           quorum: 2
         ...
 
-    Create a space called ``sync`` and enable synchronous replication on this space.
-    Then, create an index.
+    Create a space called ``sync`` and enable synchronous replication on this space:
+
+    ..  code-block:: console
+
+        app:instance001> s = box.schema.space.create("sync", {is_sync=true})
+        ---
+        ...
+
+    Then, create an index:
 
-    ..  code-block:: tarantoolsession
+    ..  code-block:: console
 
-        tarantool> s = box.schema.space.create("sync", {is_sync=true})
-        tarantool> _ = s:create_index('pk')
+        app:instance001> _ = s:create_index('pk')
+        ---
+        ...
 
     After that, use ``box.ctl.promote()`` function to claim a queue:
```

Comment:
To claim **the** queue, no? We mean specifically the transaction limbo (also known as the synchro queue).